tcms/utils/tests/test_assign_permissions.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth.models import Group, Permission
+from django.test import TestCase
+
+from tcms.utils.permissions import assign_default_group_permissions
+
+
+class TestAssignDefaultGroupPermissions(TestCase):
+    """Test utils.permissions.assign_default_group_permissions()"""
+
+    def setUp(self):
+        self.admin = Group.objects.get(name='Administrator')
+        self.admin.permissions.all().delete()
+        self.tester = Group.objects.get(name='Tester')
+        self.tester.permissions.all().delete()
+        self.all_perms_count = Permission.objects.all().count()
+        assign_default_group_permissions()
+
+    def test_assign_admin_permissions(self):
+        self.assertEqual(self.all_perms_count, self.admin.permissions.count())
```

Comment:
`all_perms_count` is not used anywhere else and is not needed as a stand alone variable. You can compare the two query expressions directly and that would be much more clear.